addons/survey/static/src/scss/survey_survey_views.scss

```diff
@@ -146,3 +146,31 @@
 .icon_rotates {
     transform: rotate(180deg);
 }
+
+/* Style of the tiles allowing the user to load a sample survey. */
+.o_survey_sample_tile {
+    position: relative;
+    width: 150px;
+    height: 150px;
+    img {
+        max-height: 125px;
+    }
+    div {
+        display: none;
+        position: absolute;
+        top: 0;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        overflow-y: auto;
+        cursor: pointer;
+    }
+    &:hover {
+        div {
+            display: flex;
+            align-items: center;
+            color: white;
```

Comment:
I think there is a font-white or color-white class or something like that.

Reply:
I would hope that "text-white" is always ... well ... white, despite any installed theme ^^